.github/workflows/integration.yml

```diff
@@ -58,6 +58,31 @@ jobs:
           currents-project-id: ${{ secrets.CURRENTS_PROJECT_ID }}
           currents-record-key: ${{ secrets.CURRENTS_RECORD_KEY }}
 
+  acceptance-tests-changed:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          fetch-depth: '2'
```

Comment:
Does it still work if the branch has more than 2 commits?

Reply:
Good point; I doubt it can work with more than that then, I've removed the parameter again.